py3status/modules/xrandr_toggle.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle monitor output from the status bar using xrandr.
+
+Configuration parameters:
+    cache_timeout: how often we refresh usage in seconds (default: 10s)
+    distext: custom text or symbol for disconnected status
```

Comment:
> FYI PEP8 has changed to explicitly allow 100 character lines now. 

I'd say it hasn't.

> Limit all lines to a maximum of 79 characters. 

there is an option

> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters)

This is one of these things, 10 years ago I'd have said it was stupid.  Now I think it is great.  In my current work flow I don't really  like lines over 60 chars.  Also I think sticking to it makes for much better programmers (though it can be frustrating to get there).  

On the subject `yapf` creates some horrible code trying to make things compliant. 

https://www.python.org/dev/peps/pep-0008/#maximum-line-length